scene/gui/color_mode.h

```diff
@@ -121,6 +127,8 @@ class ColorModeOKHSL : public ColorMode {
 public:
 	String labels[3] = { "H", "S", "L" };
 	float slider_max[4] = { 359, 100, 100, 255 };
+	float cached_hue;
```

Comment:
```suggestion
	float cached_hue = 0.0;
```
Needs to be initialized explicitly.